### ISSUE TITLE
LightGBM testing

### ISSUE THREAD

**Issue by eohara-ie** [eohara-ie/stroke-predictions#6]

**Comment by eohara-ie** [comment]
Looking for loss function to maximise F1 score in LightGBM Python API
